packages/teams-js/src/internal/mediaUtil.ts

```diff
@@ -14,11 +14,11 @@ import { throwExceptionIfMobileApiIsNotSupported } from './internalAPIs';
  * @internal
  * Limited to Microsoft-internal use
  */
-export function createFile(assembleAttachment: media.AssembleAttachment[], mimeType: string): Blob {
+export function createFile(assembleAttachment: media.AssembleAttachment[], mimeType: string): Blob | null {
   if (assembleAttachment == null || mimeType == null || assembleAttachment.length <= 0) {
     return null;
   }
-  let file: Blob | undefined;
+  let file: Blob | null = null;
```

Comment:
>let file: Blob | null = null; [](http://example.com/codeflow?start=2&length=29)

`null` is more clearly meant to indicate "none of this thing" than `undefined` is. Since the code was only "accidentally" using `undefined` before (by nature of just never setting a value), I switched it to `null` instead to make it clearly intentional. The actual behavior of this function and calling functions remains the same.